wikispeedruns/achievements/achievements.py

```diff
@@ -35,7 +35,8 @@ def convert_to_standard(raw_data: Dict[str, Any]) -> Dict[str, Any]:
     version_map = get_version_map()
     path = json.loads(raw_data["path"])
     version = path["version"]
-    return version_map[version](raw_data)
+    version_func = version_map[version] if version in version_map else version_2_1
```

Comment:
Why not add the new version to the map?

Reply:
I can if you'd like, but this will always be an issue on each db version bump. Is there a good reason we shouldn't just use this as the default (until the default needs to be changed)? @danielliu04